dotCMS/src/main/java/com/dotcms/jitsu/EventLogRunnable.java

```diff
@@ -0,0 +1,54 @@
+package com.dotcms.jitsu;
+
+import com.dotmarketing.util.Config;
+import java.util.Map;
+import com.dotcms.http.CircuitBreakerUrl;
+import com.dotcms.http.CircuitBreakerUrl.Method;
+import com.dotcms.http.CircuitBreakerUrlBuilder;
+import com.dotmarketing.util.Logger;
+import com.google.common.collect.ImmutableMap;
+
+public class EventLogRunnable implements Runnable {
+
+    final String POST_URL = Config.getStringProperty("EVENT_LOG_POSTING_URL",
```

Comment:
I don't know if here make sense have a default value, maybe we just should send a error if we don't have any values in EVENT_LOG_POSTING_URL

Reply:
This is ok for now. We can change later